src/runtime_src/driver/xclng/drm/xocl/lib/libxdma.c

```diff
@@ -3503,6 +3503,7 @@ static int pci_check_extended_tag(struct xdma_dev *xdev, struct pci_dev *pdev)
 	v =  read_register(reg);
 	v = (v & 0xFF) | (((u32)32) << 8);
 	write_register(v, reg, XDMA_OFS_CONFIG + 0x4C);
+	return 0;
```

Comment:
To avoid this kind of mistake in the future, we should enable '-Werror=return-type'.

I had a talk with Karen about this code at last friday afternoon. I will create another PR to refine this.

Reply:
This reminds me http://jira.xilinx.com/browse/CR-980617
Even if it is not the same device driver, I am still thinking we should avoid randomware and other warningware... :-)